calendar.php

```diff
@@ -596,6 +596,46 @@
 		// Redirect back to the main calendar view.
 		redirect("calendar.php", $lang->redirect_eventdeleted);
 	}
+	else
+	{
+		error($lang->delete_no_checkbox);
```

Comment:
`delete_no_checkbox` doesn't seem to be defined anywhere, at least I can't find it in my local installation with this PR.